recipe/meta.yaml

```diff
@@ -23,49 +24,67 @@ source:
   folder: dotnet
 
 build:
-  number: 2
+  number: 3
   binary_relocation: False  # [osx]
+  script_env:
+    - DOTNET_RUNTIME_VERSION={{ runtime_version }}
 
 outputs:
   - name: dotnet
     version: {{ sdk_version }}
+    build:
+      run_exports:
+        - {{ pin_subpackage("dotnet-runtime", max_pin="x.x") }}
     requirements:
       run:
-        - {{ pin_subpackage('dotnet-runtime', exact=True) }}
         - {{ pin_subpackage('dotnet-aspnetcore', exact=True) }}
         - {{ pin_subpackage('dotnet-desktop', exact=True) }}  # [win]
+        - {{ pin_subpackage('dotnet-runtime', max_pin="x.x") }}
```

Comment:
Just to confirm, this will use the version built here and not pull it from conda?

Reply:
I guess the issue I could imagine is that we release 10.0, and this framework doesn't exist yet in `conda`.

Reply:
Sorry, I just saw your comments. Yes, most of the commented section should ideally be removed in the final PR (I forgot to draft it). I was testing the problematic `linux-aarch64` that had this odd x64 ld.so. I removed it and it seems to work fine. I have this bad habit of commenting-out instead of relying on small commits and VCS to trace back

Reply:
To the specific question, I think that the pin_subpackage is to indicate that the package is built with the recipe, as opposed to pin_compatible, but I should probably research it a bit.